src/memory/combat_manager.rs

```diff
@@ -66,6 +68,36 @@ pub struct LiveMana {
     pub small: u32,
 }
 
+#[derive(Default, Debug, Clone)]
+pub struct EquippedTrinket {
+    pub trinket: Option<Item>,
+}
```

Comment:
Is this struct needed? Could just use `Option<Item>`

Reply:
Cant implement another crates trait due to orphan rule but i found out you can just wrap it :)